Article/Article.js

```diff
@@ -1,20 +1,63 @@
-// Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// Animate the article opening. Change the css. animate the component so that it slids open and slides closed on each click. Update the text "Click to Expand or "Click to Close
 
+// Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+let articleCounter = 1;
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class.
+    this.expandButton = this.domElement.querySelector(".expandButton");
+    this.deleteX = this.domElement.querySelector(".deleteX");
     // Using your expandButton reference, update the text on your expandButton to say "expand"
```

Comment:
Well written selector and assignment for the `expandButton` and `deleteX`properties.